tests/ExtraSessionTraitTest.php

```diff
@@ -32,6 +32,7 @@ public function testMaximizeWindowOnBeforeScenario(): void
         $driver->expects($this->once())->method('maximizeWindow');
         $session->expects($this->once())->method('getDriver')->willReturn($driver);
 
+        /** @var ExtraSessionTrait|MockObject $mock */
```

Comment:
updating phpdoc of `getExtraSessionMock` should work right?

Reply:
nope it does not. then phpdoc can't fit the return type (can't have multiple return type). Solution would be to drop all phpdoc as now type hitting + return type seems enough to me

Reply:
Maybe we could think about that later, what if you remove the returned type?

Reply:
it does not :/ it result in:
```
------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   tests/ExtraSessionTraitTest.php
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  149    Return typehint of method Tests\Ekino\BehatHelpers\ExtraSessionTraitTest::getExtraSessionMock() has invalid type Ekino\BehatHelpers\ExtraSessionTrait.
  151    Method Tests\Ekino\BehatHelpers\ExtraSessionTraitTest::getExtraSessionMock() should return Ekino\BehatHelpers\ExtraSessionTrait&PHPUnit\Framework\MockObject\MockObject but returns PHPUnit\Framework\MockObject\MockObject.
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

for this is the clearest way to do it at level 7